setup.cfg

```diff
@@ -1,68 +1,109 @@
 [metadata]
 name = ScoreCode
+version = 0.0.0
 license = Apache-2.0
-
-# description must be on ONE line https://github.com/pypa/setuptools/issues/1390
 description = A package to fetch data from OpenSSF Scorecard API
 long_description = file:README.rst
-long_description_content_type = text/x-rst
-url = https://github.com/nexB/ScoreCode
-
 author = nexB. Inc. and others
-author_email = info@aboutcode.org
-
+url = https://github.com/nexB/ScoreCode
 classifiers =
-    Development Status :: 5 - Production/Stable
+    Development Status :: 4 - Beta
     Intended Audience :: Developers
-    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
     Programming Language :: Python :: 3 :: Only
-    Topic :: Software Development
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
+    Topic :: Software Development :: Libraries
     Topic :: Utilities
-
+    Typing :: Typed
 keywords =
-    utilities
-
+    package
+    url
+    package manager
+    package url
 license_files =
-    apache-2.0.LICENSE
-    NOTICE
-    AUTHORS.rst
-    CHANGELOG.rst
-    CODE_OF_CONDUCT.rst
+    - mit.LICENSE
+    - AUTHORS.rst
+    - README.rst
+    - CONTRIBUTING.rst
+    - CHANGELOG.rst
 
 [options]
-package_dir =
-    =src
+python_requires = >=3.7
 packages = find:
+package_dir = =src
 include_package_data = true
 zip_safe = false
-
-setup_requires = setuptools_scm[toml] >= 4
-
-python_requires = >=3.7
-
 install_requires =
 
-
 [options.packages.find]
 where = src
 
+[options.package_data]
+packageurl-python = py.typed
+
+; [options.extras_require]
+; lint =
+;     isort
+;     black
+;     mypy
+; test =
+;     pytest
+; build =
+;     setuptools
+;     wheel
+; sqlalchemy =
+;     sqlalchemy >= 2.0.0
+
```

Comment:
We need the documentation parts you removed, we will need to have documentation too through sphinx/rtd